packages/core/src/routes/sign-in-experience.test.ts

```diff
@@ -20,6 +20,19 @@ jest.mock('@/queries/sign-in-experience', () => ({
   ),
 }));
 
+const validateBranding = jest.fn();
+const validateTermsOfUse = jest.fn();
+
+jest.mock('@/lib/validate-sign-in-experience', () => ({
+  ...jest.requireActual('@/lib/validate-sign-in-experience'),
+  validateBranding: (branding: Branding | undefined) => {
+    validateBranding(branding);
+  },
+  validateTermsOfUse: (termsOfUse: TermsOfUse | undefined) => {
+    validateTermsOfUse(termsOfUse);
+  },
```

Comment:
why do we need this? can we just expect the original function to be called with foo?

Reply:
Updated. PTAL